### PR TITLE
Updating AdSense PHP examples for v2.

### PR DESCRIPTION
**Pull request by donovanfm**

### CODE REVIEW

File: v2/php/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* Change the include path in adsense-sample.php to your vendor directory if
+  needed.
+* Modify `client_secrets.json` with your client ID, client secret and redirect
+  URL (`http://your/path/adsense-sample.php`).
+* (Optional) If you want to store credentials between runs to avoid authorizing
+  more than once, change `STORE_ON_DISK` in adsense-sample.php to `true`.
+  * You may have to give your PHP installation write permissions to the token
+    file. One easy way of doing this is creating an empty `tokens.dat` file in
+    the installation directory and making it writeable by your web server.
+* Open the sample (`http://your/path/adsense-sample.php`) in your browser.
```
Comment (abadreldin) [inline v2/php/README.md:30]: Suggest adding `http://localhost:8000/adsense-sample.php` as an option to reduce any potential confusion.

File: v2/php/adsense-sample.php [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+// Create service.
+$service = new Google_Service_Adsense($client);
+
+// Logging out will remove the current access token and delete the token file
+// if a token file exists (i.e., STORE_ON_DISK is enabled).
+if (isset($_REQUEST['logout'])) {
+  unset($_SESSION['access_token']);
+  unlink(TOKEN_FILENAME);
+}
+
```
Comment (abadreldin) [inline v2/php/adsense-sample.php:79]: Added ```// Configure redirect uri to return to main page once the authentication workflow has completed.
$redirectUri = 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['PHP_SELF'];
$client->setRedirectUri($redirectUri);` to enable redirect.

File: v2/php/examples/GenerateSavedReport.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+class GenerateSavedReport {
+  /**
+   * Retrieves a report for the specified ad client.
+   *
+   * @param $service Google_Service_AdSense AdSense service object on which to
+   *     run the requests.
+   * @param $savedReportId string the saved report ID on which to run the
+   *     report.
+   */
+  public static function run($service, $savedReportId) {
```
Comment (abadreldin) [inline v2/php/examples/GenerateSavedReport.php:32]: I believe this requires account ID.
```suggestion
  public static function run($service, $savedReportId, $accountId) {
```

File: v2/php/adsense-sample.php [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+    } else {
+      print "No ad clients found, unable to run dependent examples.\n";
+    }
+
+    $savedReports = GetAllSavedReports::run($service, $exampleAccountId,
+        MAX_LIST_PAGE_SIZE);
+    if (isset($savedReports) && !empty($savedReports)) {
+      // Get an example saved report ID, so we can run the following sample.
+      $exampleSavedReportId = $savedReports[0]['name'];
+      GenerateSavedReport::run($service, $exampleSavedReportId);
```
Comment (abadreldin) [inline v2/php/adsense-sample.php:193]:
```suggestion
      GenerateSavedReport::run($service, $exampleSavedReportId, $exampleAccountId);
```

File: v2/php/adsense-sample.php (hunk 2 above)
Comment (donovanfm) [reply v2/php/adsense-sample.php:79]: Can you help me understand exactly where this is needed? In the logout conditional?

File: v2/php/examples/GenerateSavedReport.php (hunk 3 above)
Comment (donovanfm) [reply v2/php/examples/GenerateSavedReport.php:32]: Actually the accountId isn't needed, and it was just a leftover print statement. PHP is really lenient about missing variables, so it just printed an empty string. Anyway, I've removed it in GenerateSavedReport.php.

**Review by abadreldin — APPROVED** [review]
LGTM

**Pull request merged**